sdk/servicebus/azure-messaging-servicebus/CONTRIBUTING.md

```diff
@@ -1,92 +0,0 @@
-# Contributing
```

Comment:
This reminds me, I should remove `CONTRIBUTING.md` from Event Hubs too. I will do it when I update the versions for May release.